apps/console/src/public/deployment.config.json

```diff
@@ -215,6 +215,10 @@
         "appName": "Console",
         "appLogoPath": "/assets/images/branding/logo.svg",
         "appWhiteLogoPath": "/assets/images/branding/logo-white.svg",
+        "emailTemplates": {
+            "defaultLogoUrl": "https://wso2.cachefly.net/wso2/sites/all/2023/images/wso2-logo.png",
```

Comment:
We need to get these from the theme otherwise default logo will not be visible when running the server offline.

Reply:
Fixed with 9689f92